src/MCPClient/lib/clientScripts/create_mets_v2.py

```diff
@@ -1141,6 +1240,16 @@ def createFileSec(
                         f.originallocation.replace("%transferDirectory%", "", 1),
                         state,
                     )
+                    XMLDMDID = createDMDFromXML(
```

Comment:
I had some conflicts about this ... I don't think that pattern represents the actual purpose of those functions. I also thought about renaming the other one and including this call inside it, since the params are the same in the end, but I don't want to mess a lot with existing functions.